packages/block-editor/src/components/inserter/test/block-types-tab.native.js

```diff
@@ -1,67 +0,0 @@
-/**
- * External dependencies
- */
-import { render } from 'test/helpers';
-
-/**
- * WordPress dependencies
- */
-import { useSelect } from '@wordpress/data';
-
-/**
- * Internal dependencies
- */
-import items from './fixtures';
-import BlockTypesTab from '../block-types-tab';
```

Comment:
Without having had a look at the Native code for this component, why is this test being removed? I wasn't sure how it relates to the rest of the PR, or is it just a code cleanup?

Reply:
The test was failing and then I looked into fixing it and it involves mocking too much IMO and the test itself doesn't do much. Do you see any value in this test?